src/compiler/diagnosticMessages.json

```diff
@@ -751,8 +751,22 @@
         "category": "Error",
         "code": 1235
     },
-
-
+    "'{0}' modifier can only appear on a class or member function declaration.": {
+        "category": "Error",
+        "code": 1236
+    },
+    "'{0}' modifier cannot be used with '{1}' modifier.": {
+        "category": "Error",
+        "code": 1237
+    },
+    "Abstract methods can only appear within an abstract class.": {
+        "category": "Error",
+        "code": 1238
+    },
+    "Abstract member functions cannot have an implementation.": {

```

Comment:
how about "method '{0}' can not have an implantation because it is marked abstract"